src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,59 @@
+    <mdb-navbar SideClass="navbar navbar-expand-xl navbar-light white scrolling-navbar">
+        <logo>
+          <a class="navbar-brand" href="#">
+            <img src ="/assets/img/logo.png">
+          </a>
+      </logo>
+      <links>
+      <ul class="navbar-nav mr-auto">
+          <li class="nav-item bold">
+            <a class="nav-link waves-light active navl" mdbWavesEffect routerLink="">Home <span class="sr-only">(current)</span></a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link waves-light navl" mdbWavesEffect routerLink="/about">About Us</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link waves-light navl" mdbWavesEffect routerLink="/projects">Projects</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link waves-light navl" mdbWavesEffect routerLink="/contact">Contact Us</a>
+        </li>
+    </ul>
+    
+    <!--Contact Us Social Media Icons-->
+    <ul class="navbar-nav nav-flex-icons">
+      <li class="nav-item waves-light" mdbRippleRadius>
+        <!--Facebook-->
+        <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://www.facebook.com/AnitaB.0rg/"><i class="fa fa-facebook fa-2x"></i></a>
+    </li>
+    <li class="nav-item waves-light" mdbRippleRadius>
+        <!--Twitter-->
+        <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://twitter.com/systers_org"><i class="fa fa-twitter fa-2x"></i></a>
+    </li>
+    <li class="nav-item waves-light" mdbRippleRadius>
+        <!--YouTube-->
+        <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://www.youtube.com/channel/UCUvti1DEKYUZBtQqQAtCK6g/videos"><i class="fa fa-youtube fa-2x"></i></a>
+    </li>
+    <li class="nav-item waves-light" mdbRippleRadius>
+        <!--Medium-->
+        <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://medium.com/@systers_org"><i class="fa fa-medium fa-2x"></i></a>
+    </li>
+    </ul>
+    <!--End of Contact Us Social Media Icons-->
+    </links>
+    </mdb-navbar>
+
+    <script>
```

Comment:
this part does not seem to work.
Also it is not good practice to include script in html file.